legacy/tests/integration/fundTrading.js

```diff
@@ -55,13 +55,22 @@ test.before(async t => {
 
   const [r, s, v] = await getTermsSignatureParameters(manager);
   await deployed.FundFactory.methods.createComponents(
-    'Test Fund', [deployed.MatchingMarket.options.address], [deployed.MatchingMarketAdapter.options.address], deployed.EthToken.options.address, [deployed.EthToken.options.address, deployed.MlnToken.options.address], [false], deployed.TestingPriceFeed.options.address
+    'Test Fund',
+    [deployed.MatchingMarket.options.address],
+    [deployed.MatchingMarketAdapter.options.address],
+    deployed.EthToken.options.address,
+    [deployed.EthToken.options.address, deployed.MlnToken.options.address],
+    [false],
+    deployed.TestingPriceFeed.options.address
   ).send({from: manager, gasPrice: config.gasPrice});
   await deployed.FundFactory.methods.continueCreation().send({from: manager, gasPrice: config.gasPrice});
   await deployed.FundFactory.methods.setupFund().send({from: manager, gasPrice: config.gasPrice});
   const fundId = await deployed.FundFactory.methods.getLastFundId().call();
   const hubAddress = await deployed.FundFactory.methods.getFundById(fundId).call();
   fund = await getFundComponents(hubAddress);
+  await Promise.all(Object.values(fund).map(async (component) => {
+    await deployed.MockVersion.methods.setIsFund(component.options.address).send({from: manager});
```

Comment:
This sets isFund for all fund components? Why?

Reply:
The `isFund` query on `MockVersion` is dumb and only tracks everything in a mapping. When we make the actual version contract we will have to change it a bit so that components (which are not funds themselves) can pay amgu. May also need a rename at that point. Anyway yeah this is just there to pass an assertion when paying amgu in tests at the moment